src/main/webapp/search-meal-script.js

```diff
@@ -46,7 +46,7 @@ function createMealElement(dish) {
   const aElement = document.createElement("a");
   aElement.setAttribute("href", `meal.html?id=${id}`);
   const insideDivElement = document.createElement("div");
-  insideDivElement.appendChild(createElementByTag(title, "b"));
+  insideDivElement.appendChild(createElementByTag(capitalizeFirst(title), "b"));
```

Comment:
I guess I found where problem with encoding appears.
I guess we will need to add `encodingCheck()` here too as I did it in `meal-page-script.js`. 
I made PR to add this func to `script.js` file.